nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -522,8 +544,9 @@ def quantize_output(self, output, wrapper, **kwargs):
                                                                        module.ema_decay)
             module.tracked_max_output = update_ema(module.tracked_max_output, current_max,
                                                                        module.ema_decay)
-            module.scale, module.zero_point = update_quantization_param(
-                output_bits, module.tracked_min_output, module.tracked_max_output)
+        module.scale, module.zero_point = update_quantization_param(
```

Comment:
a little confused that we use the same var `module.scale, module.zero_point` in quantizing weight, input, output

Reply:
seems indeed have problem here, at least error-prone. and we should think about what kind of information should we export, or how to easily get scale / zero_point from user's point of view

Reply:
Yes, it is better to define stand-alone scale & zero_point for each quant_type. I will upload a refactor pr about this when this pr is merged.